### DISCUSSION TITLE
Display of accents or special characters for languages ​​other than English

### DISCUSSION THREAD

**Discussion by martin66789** [mriscoc/Ender3V2S1#807] [Q&A]
I upgraded the Hungarian translation of the firmware to the current version, and it works, but there are characters that the display does not show. These would be the letter characters: "ö, Ö, ü, Ü, ó, Ó, ő, Ő, ú, Ú, é, É, á, Á, ű, Ű, í, Í".

I tried the original and the Giadej compilation display firmware, but neither of them displayed the characters listed above.

Giadej compilation display firmware font.ttf file contains the characters in question, why does the printer not display them?
Or what else should be changed to make the characters appear?

I have an Ender 3 V2 printer with Ender 3S1 DACIA display.

Thanks in advance for your help.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/issues/642

**Reply by martin66789** [reply]
I understand. I've solved it by using the correct letter without accents instead of the accented letters.
While checking the translation on the printer, I found some text that were not translated (missing from the language_en.h file). Where can I find these?

"Move Ext. 100mm"
"WARNING: current position isunknown, home axes.."
"Move Z to Home"
"Preheat CUSTOM"
"Preheat CUSTOM Conf"
"Hotend PID Settings"
"Hotend PID"
"Set Kp:"
"Set Ki":
"Set Kd:"
"Bed PID Settings"
"Bed PID"
"Set Kp:"
"Set Ki":
"Set Kd:"
"End-stops diag."
"End-stop Diagnostic"
"x_min"
"Y_min"
"filament"
"open"
"TRIGGERED"
"Please confirm"
"115K baud"
"Screen Background"
"Cursor"
"Title Background"
"Title Text"
"Selected"
"Split Line"
"Highlight"
"Status Background"
"Status Text"
"Popup Background"
"Popup Text"
"Alert Background"
"Alert Text"
"Percent Text"
"Bar Fill"
"Indicator Value"
"Coordinate Value"
"Calculating average and relative heights"
"Corners not leveled"
"Knob adjustment required"
"Lower"

**Reply by mriscoc** [reply]
Some are in the dwin.cpp file, for others you can use the search function in VSCode

**Reply by martin66789** [reply]
Where can I find the texts at the top of the picture? I found all the others, but not these yet.
https://i.imgur.com/jN2VD8F.jpg

**Reply by mriscoc** [reply]
They are in the header of the G-code file and hard coded in the library.
